src/MaxText/rl/evaluate_rl.py

```diff
@@ -85,18 +116,41 @@ def generate_responses(
   return multiple_call_responses
 
 
+def _to_float(value: str) -> float:
+  """Convert numeric strings honoring local formatting."""
+  sanitized = value.strip().replace("\u00A0", " ").replace("\u202F", " ")
+  _ensure_locale_numeric()
```

Comment:
I understand the need to replace comma with empty string, but for the locale specific changes where do you see issues with the current implementation ?

Reply:
Removed changes in evalutate